src/main/content/blog_tags.json

```diff
@@ -54,7 +54,16 @@
         {
             "name": "community",
             "posts": ["open-liberty-dev-mode", "java-microprofile-jwt-okta"]
+        },
+        {
+            "name": "vscode",
+            "posts": ["liberty-dev-mode-vscode"]
```

Comment:
I'm not sure if we want to have a tag with only one post. It might look a bit odd and I'm wondering if we'll have many other posts related to VS Code